packages/breadcrumb/src/js/breadcrumb.js

```diff
@@ -0,0 +1,48 @@
+"use strict";
+
+/**
+ * Breadcrumb module
+ * @file Handles interaction behaviour for the breadcrumb.
+ */
+
+class Breadcrumb {
+  constructor(el) {
+    this.init(el);
+  }
+
+  init(el) {
+    const breadcrumbs = el.querySelectorAll(".uq-breadcrumb__item");
```

Comment:
This could be moved to the constructor and be something like:
```
// The breadcrumb trail.
this.breadcrumbs = el.querySelectorAll(".uq-breadcrumb__item");
```